apps/institution-service/src/controllers/aggregatorIntegrationController.ts

```diff
@@ -38,3 +42,30 @@ export const updateAggregatorIntegration = async (
     });
   }
 };
+
+export const createAggregatorIntegration = async (
+  req: Request,
+  res: Response,
+) => {
+  try {
+    const aggregatorIntegration = await AggregatorIntegration.create(
+      req.body as AggregatorIntegrationParams,
+    );
+
+    return res.status(201).json({
+      message: "AggregatorIntegration created successfully",
+      aggregatorIntegration,
+    });
+  } catch (error) {
+    if (error instanceof ValidationError) {
+      res.status(400).json({
+        error: "Invalid aggregatorIntegration Data",
+        message: error.errors[0]?.message,
+      });
+    } else if (error instanceof Error) {
+      res.status(400).json({ error: "Database Error", message: error.message });
+    } else {
+      res.status(500).json({ error: "Error", message: "something went wrong" });
```

Comment:
This is a nitpick, and not that important, but I'd use sentence case in any error message, just for consistency. "Something went wrong"

Reply:
Agreed